<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>root_win</class>
 <widget class="QWidget" name="root_win">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>管理员登录</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 14pt &quot;Adobe Arabic&quot;;</string>
  </property>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>账号：</string>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>210</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>密码：</string>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="line1">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>210</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="line3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>290</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_3">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>20</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
